audit/gather/gather.go

```diff
@@ -205,7 +205,7 @@ func (g *gather) start() error {
 	if g.cfg.Include.ServerEndpoints {
 		err := g.captureServerEndpoints(serverInfoMap, g.cfg.Detailed)
 		if err != nil {
-			return fmt.Errorf("failed to capture server endpoints")
+			return fmt.Errorf("failed to capture server endpoints: %w", err)
```

Comment:
Should this be 
`g.log.Errorf`, so we do not stop and still try to collect as much as possible? I am happy to do that in a separate PR?

Also, all the `return fmt.Errorf` calls in this function only print logs to console output, and we will miss them in `capture/misc/audit_gather_log.log` in the shared archive.